src/pages/createpin.css

```diff
@@ -0,0 +1,145 @@
+h1, h2, h3, h4, h5, h6,p{
```

Comment:
we can use tailwind.css

Reply:
refrain from adding class to generic documents if it is not aplied through use classes